web-client/src/presenter/computeds/selectDocumentTypeHelper.test.ts

```diff
@@ -48,16 +49,28 @@ describe('selectDocumentTypeHelper', () => {
         scenario: 'Nonstandard A',
       };
 
+      const mockDocuments = MOCK_DOCUMENTS.map(d => {
+        return {
+          ...d,
+          isOnDocketRecord: true,
+          isStricken: false,
+          servedAt: '2019-08-25T05:00:00.000Z',
+        };
+      });
```

Comment:
`MOCK_DOCUMENTS` is used in a lot of places. I modified them here in the tests to avoid affecting other tests.